### PR TITLE
Changelog update for v0.7.2

### PR DESCRIPTION
**Pull request by ivanprado**

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,13 @@
 History
 =======
 
+v0.7.2 (TBA)
+------------
+
+* ignore any exception if errors='ignore'
```
Comment (kmike) [inline HISTORY.rst]: hey! would you mind making it a bit more detailed, e.g. which function is this argument for?

Comment (ivanprado) [reply HISTORY.rst]: @kmike hope now is more clear.

File: HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,13 @@
 History
 =======
 
+v0.7.2 (TBA)
+------------
+
+* Cover all possible exception cases dealt by ``extruct()`` ``errors`` attribute for values ``strict``, ``log`` and ``ignore``
+* avoid including ``itemprop`` from child ``itemscope`` when using ``itemref``
+* proper processing order for ``itemref``
```
Comment (kmike) [inline HISTORY.rst]: Would you mind mentioning which extractors / semantic markup formats it affects?

Comment (ivanprado) [reply HISTORY.rst]: Sure @kmike. Done!

**Comment by kmike** [comment]
Thanks @ivanprado!

**Pull request merged**